### PR TITLE
Dockerised the nextapp

### PR DESCRIPTION
**Pull request by Avijit2002**
Use "docker build . -t image_name" to build the image.
Use "docker run  -d --env-file <env_file_name> -p 3000:3000 image_name" to run the image.

### CODE REVIEW

**Comment by zeul22** [comment]
Hi, the code seems a little off as we have not used 'src/' directory. Also would you be able to set up env-variables using docker-compose that takes values from .env file?

File: DockerFile
Comment (zeul22) [inline DockerFile:1]: Please modify the WorkDIR and add env variables referenced from docker-compose

Comment (Avijit2002) [reply DockerFile:1]: okk sure

**Comment by Avijit2002** [comment]
Added compose.yml file and fixed the working directory path.

File: compose.yml
Comment (zeul22) [inline compose.yml:1]: Please rename the file as 'docker-compose.yml'

**Review by zeul22 — CHANGES_REQUESTED** [review]

File: compose.yml
Comment (Avijit2002) [reply compose.yml:1]: okk

**Comment by Avijit2002** [comment]
compose.yml file renamed as 'docker-compose.yml.

File: compose.yml
Comment (Avijit2002) [reply compose.yml:1]: Done

**Comment by zeul22** [comment]
Thank you Avijit for the contribution, there are other issues as well if you are interested!

**Comment by Avijit2002** [comment]
Thank you @zeul22 for merging my PR. Will surely contribute more to this project.

**Pull request merged**